src/main/java/HomeScreenView.java

```diff
@@ -1,43 +1,141 @@
+    /**
+     * The input location (Postal Code)
+     */
+    JTextField location;
+    /**
+     * The selected price bucket (pricing /10)
+     */
+    JComboBox<Integer> priceBucket;
+    /**
+     * The select star rating (/5 stars)
+     */
+    JComboBox<Integer> avgStars;
+    /**
+     * The selected cuisine type
+     */
+    JComboBox<String> cuisineType;
+    /**
+     * The group of radiobuttons for the sorting selection
+     */
+    ButtonGroup sortButtons;
+    /**
+     * The choices controller
+     */
+    ChoicesController choicesController;
 
-    // Search Field(s) Temporarily placing Location as an input field
-    JTextField searchBox = new JTextField();
-    JTextField locationBox = new JTextField();
+    public HomeScreenView(ChoicesController choicesController) {
 
-    // Drop-Down Menus Options
-    Integer[] pricingOptions = { 1, 2, 3, 4, 5, 6, 7, 8, 9, 10};
-    Integer[] avgStarsOptions = { 1, 2, 3, 4, 5};
+        this.choicesController = choicesController;
 
-    // TODO: add more cuisine options
-    String[] cuisineOptions = { "Food"};
+        // Search Field(s) Temporarily placing Location as an input field
+        JPanel searchField = new JPanel();
 
+        query = new JTextField();
+        location = new JTextField();
 
-    // Drop-Down Menus
-    JComboBox pricingDropDown = new JComboBox(pricingOptions);
-    JComboBox avgStartsDropDown = new JComboBox(avgStarsOptions);
-    JComboBox cuisineDropDown = new JComboBox(cuisineOptions);
+        searchField.add(query);
 
-    // Sort Method Buttons
-    JRadioButton sortPriceButton = new JRadioButton();
+        // Search Button
+        JButton searchButton = new JButton("Search");
+        searchField.add(searchButton);
+        searchButton.addActionListener(this);
 
-    JRadioButton sortAvgStarsButton = new JRadioButton();
+        // Drop-Down Menus Options
+        Integer[] pricingOptions = { 1, 2, 3, 4, 5, 6, 7, 8, 9, 10};
+        Integer[] avgStarsOptions = { 1, 2, 3, 4, 5};
 
-    JRadioButton sortNameButton = new JRadioButton();
+        // TODO: add more cuisine options
+        String[] cuisineOptions = {"Food"};
 
-    ChoicesController choicesController;
 
-    public HomeScreenView(ChoicesController choicesController) {
+        // Drop-Down Menus
+        priceBucket = new JComboBox(pricingOptions);
+        avgStars = new JComboBox(avgStarsOptions);
+        cuisineType = new JComboBox(cuisineOptions);
+
+        // Filter Options
+        JPanel filterFields = new JPanel();
+        filterFields.add(priceBucket);
+        filterFields.add(avgStars);
+        filterFields.add(cuisineType);
+
+
+        // Sort Method Buttons
+        JPanel sortFields = new JPanel();
+        ButtonGroup sortButtons = new ButtonGroup();
+
+        // Initialize Buttons
+        JRadioButton sortPriceButton = new JRadioButton();
+        JRadioButton sortAvgStarsButton = new JRadioButton();
+        JRadioButton sortNameButton = new JRadioButton();
+
```

Comment:
Suggestion: add another radio button for order, for instance if they wanted to sort by price low to high vs high to low

Reply:
Noted. Will look to implement this in the interactor as well.